### PR TITLE
Makes a small adjustment to the ash walker camp.

### PR DESCRIPTION
**Pull request by BlackMajor**
-Places a safe in the Ash Walker camp, containing the malf_upgrade tech disk, and a tech disk containing the illegal techweb node.

You know funny thing is that I was supposed to do this six months ago.

### CODE REVIEW

**Comment by Seris02** [comment]
maybe also put a stethoscope in there, if there isn't one already?

File: code/modules/research/research_disk.dm [hunk 1]
```diff
@@ -20,3 +20,12 @@
 /obj/item/disk/tech_disk/debug/Initialize()
 	. = ..()
 	stored_research = new /datum/techweb/admin
+
+/obj/item/disk/tech_disk/illegal
```
Comment (Poojawa) [inline code/modules/research/research_disk.dm:24]: Why should mining get free access to unlocking antag gear?

**Review by Poojawa — CHANGES_REQUESTED** [review]
Use map merger.

File: code/modules/research/research_disk.dm (hunk 1 above)
Comment (BlackMajor) [reply code/modules/research/research_disk.dm:24]: Pooj there is literally a black and red toolbox in the exact same room that unlocks illegal tech.

Infact now that I say that I question why I even added the illegal disk if there's illegal tech right next to it.

**Comment by BlackMajor** [comment]
Fuck I forgot the map merger.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**